test/scenarios/attestation/output/src/attestation/azext_attestation/tests/latest/depSto.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Is this file duplicate with test/scenarios/attestation/input/test-scenarios/depSto.json ? I wonder why we need to put it in the generated files

Reply:
not duplicated.

The test/scenarios/attestation/input/test-scenarios/depSto.json is in swagger repo, while tests/latest/depSto.json is in generated code.
The armTemplate need to be invoked when running test, so need to be added in generated code.